### PR TITLE
Linux building

### PR DESCRIPTION
**Pull request by sanguinariojoe**
The reference guide is not implemented yet, since it uses the Microsoft's HTML Help Compiler

### CODE REVIEW

File: make-docs.sh [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+
+# Time to stop execution if something fails
+set -e
+
+# Save the working folder
+cwd=$(pwd)
+
+if [ "$BUILD_THEORY" = true ] ; then
+    cd $cwd/src/TheoryGuide
+    echo "Generate the Fedem Theory Guide (pdf^)"
```
Comment (kmoks) [inline make-docs.sh]: Can remove the "^" character here. It is needed in the BAT-file only where it escapes the following ")".

File: make-docs.sh [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+    if [ "$INSTALL" = true ] ; then
+        echo "Installing FedemTheoryGuide.pdf in $INSTALL_DIR"
+        mv FedemTheoryGuide.pdf $INSTALL_DIR/FedemTheoryGuide.pdf
+    fi
+    cd $cwd
+fi
+
+if [ "$BUILD_USER" = true ] ; then
+    cd $cwd/src/UsersGuide
+    echo "Generate the Fedem User Guide (pdf^)"
```
Comment (kmoks) [inline make-docs.sh]: Here also, remove the "^".

**Review by kmoks — CHANGES_REQUESTED** [review]
Thanks for providing the shell script. Haven't tried it myself, but looks good, except for the two ^-characters as noted (you may force-push those changes to your branch before we merge).

File: make-docs.sh (hunk 1 above)
Comment (sanguinariojoe) [reply make-docs.sh]: Done in [51dabf6](https://github.com/SAP/fedem-docs/pull/1/commits/51dabf690957ba2743274ab675cbf108e9bc877b)

File: make-docs.sh (hunk 2 above)
Comment (sanguinariojoe) [reply make-docs.sh]: Done in [51dabf6](https://github.com/SAP/fedem-docs/pull/1/commits/51dabf690957ba2743274ab675cbf108e9bc877b)

**Review by kmoks — APPROVED** [review]

**Pull request merged**